infra/README.md

```diff
@@ -14,6 +14,21 @@ Instructions:
   service account name `terraform`.  Create a JSON service account key
   and place it in `$HOME/.hail/terraform_sa_key.json`.
 
+  ```
+  gcloud iam service-accounts create terraform --display-name="Terraform Account"
+  gcloud projects add-iam-policy-binding hail-vdc --member='serviceAccount:terraform@<project-id>.iam.gserviceaccount.com' --role='roles/owner'
+  gcloud iam service-accounts keys create $HOME/.hail/terraform_sa_key.json  --iam-account=terraform@<project-id>.iam.gserviceaccount.com
+  ```
+
+  When you are finished using terraform, we recommend deleting both the service account and the key
+  file.
```

Comment:
I'm not sure I recommend that.  This wasn't intended to be 1-shot, but we eventually need to get things set up so changes to the terraform rules get applied automatically, so you'll need to keep around the terraform service account and Terraform state (which is created by `terraform apply`)